### DISCUSSION TITLE
Tencent and Amap integrate into geopy/geocoder

### DISCUSSION THREAD

**Discussion by Zeroto521** [geopy/geopy#521] [Ideas]
I would like to add Tencent ([official website](https://map.qq.com/), [wiki](https://en.wikipedia.org/wiki/Tencent_Maps)) and Amap ([official website](https://amap.com/), [wiki](https://zh.wikipedia.org/wiki/%E9%AB%98%E5%BE%B7%E5%9C%B0%E5%9C%96)) map service to geocoder.

Based on the following reasons:

- Most map services are blocked in China, related to #56
- [Restrictions on geographic data in China](https://en.wikipedia.org/wiki/Restrictions_on_geographic_data_in_China).
- The CRS of the Baidu map service is not `EPSG:4326`

**Comment by KostyaEsmukov** [comment]
You're welcome to start with a PR :)
